### PR TITLE
Add existing OpenCost plugin to the Backstage Plugin Directory

### PR DESCRIPTION
**Pull request by mattray**
The OpenCost plugin was already in Backstage, just not listed in the Plugin Directory.

### CODE REVIEW

File: microsite/static/img/opencost.png
Comment (tudi2d) [inline microsite/static/img/opencost.png:1]: You probably want to have a squared logo - maybe just the pig. I would expect the current picture to look weirde [here](https://backstage.io/plugins/):)

File: microsite/data/plugins/opencost.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: OpenCost
+author: Matt Ray
+authorUrl: https://opencost.io
+category: Monitoring
```
Comment (tudi2d) [inline microsite/data/plugins/opencost.yaml:5]: "Kubecost" & "Cost Insights" are in "Discovery" - I'm happy with "Monitoring" think it makes sense:)

**Review by tudi2d — CHANGES_REQUESTED** [review]
Great! Data looks good - maybe adjust the logo dimensions for the store.

File: microsite/static/img/opencost.png
Comment (mattray) [reply microsite/static/img/opencost.png:1]: Updated with the squared pig :)

File: microsite/data/plugins/opencost.yaml (hunk 1 above)
Comment (mattray) [reply microsite/data/plugins/opencost.yaml:5]: I'd prefer "Monitoring" since we're only reporting costs with different views.

**Review by tudi2d — APPROVED** [review]
looks good - let's go! 🚀

**Review by freben — APPROVED** [review]

**Pull request merged**